library/fluid/fluid_extensions/src/main/java/org/quiltmc/qsl/fluid/mixin/LavaMixin.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.fluid.mixin;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.fluid.FlowableFluid;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.fluid.LavaFluid;
+import net.minecraft.loot.LootTables;
+import net.minecraft.particle.ParticleEffect;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.util.random.RandomGenerator;
+import net.minecraft.world.World;
+import net.minecraft.world.event.GameEvent;
+import org.quiltmc.qsl.fluid.api.FlowableFluidExtensions;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(LavaFluid.class)
+public abstract class LavaMixin extends FlowableFluid implements FlowableFluidExtensions {
+	private static final int lavaColor = 0x991900;
```

Comment:
```suggestion
	private static final int LAVA_FOG_COLOR = 0x991900;
```
Why isn't this constant in `FlowableFluidExtensions`?

Reply:
Also, this mixin should probably be named `LavaFluidMixin`.